client_plugin/Makefile

```diff
@@ -319,6 +322,8 @@ deploy-vm-test:
 	$(DEPLOY_VM_TEST_SH) "$(VMS)" $(BIN) $(SCRIPTS)
 deploy-vm-plugin:
 	$(DEPLOY_VM_PLUGIN_SH) "$(VM2)" $(PLUGIN_BIN) $(MANAGED_PLUGIN_LOC) $(SCRIPTS) $(DOCKER_HUB_REPO) $(VERSION_TAG) $(EXTRA_TAG)
+deploy-plugin-windows-vm:
```

Comment:
@venilnoronha while looking carefully at `deploy-vm-plugin` and `deploy-plugin-windows-vm` name and underlying implementation, they are not in sync .. both targets are building the plugin by copying required source file at the targeted VMs we should rename as shown below to avoid future confusion.

 `deploy-vm-plugin` => `build-plugin`  (managed plugin for linux distros)
`deploy-plugin-windows-vm` => `build-windows-plugin`